var/spack/repos/builtin/packages/acl/package.py

```diff
@@ -23,14 +27,33 @@ class Acl(AutotoolsPackage):
     depends_on("autoconf", type="build")
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
-    depends_on("attr")
+    depends_on("bash", type="build")
```

Comment:
```suggestion
```

We can remove the dependency on bash if we instead execute the `./autogen.sh` script directly.